google_fonts/test/font_methods_test.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:google_fonts/src/google_fonts_base.dart';
+import 'package:mockito/mockito.dart';
+import 'package:http/http.dart' as http;
+
+class MockHttpClient extends Mock implements http.Client {}
+
+main() {
+  setUp(() async {
+    httpClient = MockHttpClient();
+    when(httpClient.get(any)).thenAnswer((_) async {
```

Comment:
Do these tests need a MockHttpClient?

Reply:
No, it doesn't seem to need the mock client. 

I'm embarrassed to admit that I'm not sure how to be 100% certain that the tests aren't making real life http requests without the mock client though. 

I turned off my wifi and the tests still pass, so I guess that's good enough.